src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/aat/SaveClaimTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.cmc.claimstore.aat;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.TestPropertySource;
+import uk.gov.hmcts.cmc.claimstore.BaseSaveTest;
+import uk.gov.hmcts.cmc.claimstore.RestTestClient;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.ClaimData;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@TestPropertySource(
```

Comment:
turning feature toggles off in functional tests probably doesn't do anything?

Reply:
Remove those 👌

Reply:
Actually they disable CCD and DM when tests are run in mocked mode.